src/datepicker/datepicker.js

```diff
@@ -174,7 +174,7 @@ export default class Datepicker extends React.Component<
     const inputValue = event.currentTarget.value;
 
     if (
-      inputValue === this.getMask().replace(/9/g, ' ') ||
+      inputValue === this.getMask().replace(/ /g, ' ') ||
```

Comment:
Is this change intentional? I think this condition supposed to check if the input is empty. With a mask `9999/99/99` the input